tools/install.sh

```diff
@@ -145,7 +145,8 @@ function sync_dependencies {
 			libpng-devel postgresql-devel python3-devel python3-pip \
 			libffi-devel openssl-devel
 		if [ -z "$KING_PHISHER_SKIP_CLIENT" ]; then
-			dnf install -y geos geos-devel gtksourceview3 vte3 gobject-introspection-devel
```

Comment:
Did you test that removing `vte3` doesn't break the ability to install on older versions of Fedora?

Reply:
yup this is why `vte3` is on its own line. This allows all the other required packages for 29 to be installed, and then an attempt to install `vte3` in case it is an older version of Fedora.

Reply:
So it has to be on it's own line? Is it not available in Fedora 29?

Reply:
correct

Reply:
Okay can you note that then so we know that on `Fedora 29+` that may fail and it's fine. Is it fine though, can we still use the RPC terminal or is there a newer version of the package that we should be using?

Reply:
I'll add a comment in the file. Not sure what package it is for Fedora 29+ but RPC Terminal is working just fine.

![image](https://user-images.githubusercontent.com/9035933/48096697-e2955d00-e1e5-11e8-8a33-668580df4206.png)